sw/host/cryptotest/testvectors/parsers/wycheproof_ecdsa_parser.py

```diff
@@ -54,8 +59,11 @@ def parse_test_vectors(raw_data):
                 "curve": EC_NAME_MAPPING[key["curve"]],
                 "hash_alg": group["sha"].lower(),
                 "message": list(bytes.fromhex(test["msg"])),
-                "qx": int(key["wx"], 16),
-                "qy": int(key["wy"], 16),
+                # These values are encoded as a string so the test harness can
+                # extract the entire value (serde_json likes to turn large
+                # integers into limited-precision floating point values)
```

Comment:
Nit: this comment is a bit confusing without additional context. Additionally, serde_json is not necessarily the consumer of this test vector. Recommend saying something like the following:

```python
# Encode qx and qy as a string because downstream test vector
# consumers may incorrectly handle large integers.
```